memoryGame/instruction.js

```diff
@@ -106,7 +109,7 @@ function startScore(){
 scoreId = inGameScore.innerHTML = `SCORE:10000`
 score = score - 5;
 scoreId = setInterval(function() {
-    console.log(inGameScore.innerHTML = `SCORE:${score}`);
+    inGameScore.innerHTML = `SCORE:${score}`;
```

Comment:
Very nice. I have witnessed that change with my own eyes. Probably not worth having any important calculations inside of a console.log() so this is good.